MainDemo.Wpf/App.xaml

```diff
@@ -24,7 +24,7 @@
                 <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Accent/MaterialDesignColor.Lime.xaml" />
                 -->
 
-                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" />
+                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign2.Defaults.xaml" />
```

Comment:
For now, lets leave the old Material Design Defaults dictionary named "MaterialDesignTheme.Defaults.xaml" but leave a comment that it will be renamaed in the 5.0.0 release.

Reply:
ok, I only copied and renamed the Defaults xaml. The original still exists atm for this purpose.  Do you want me to remove the references to the MaterialDesign2.Defaults.xaml file?

Reply:
I think for now, yes let's remove them. Just trying to avoid any bugs slipping in because one file gets updated and the other missed.

Reply:
I'll review this tonight and make a commit. I'll also look into the build fail above.